x-pack/docs/en/security/authentication/configuring-kerberos-realm.asciidoc

```diff
@@ -46,6 +46,28 @@ For more information on Java GSS, see
 https://docs.oracle.com/javase/10/security/kerberos-requirements1.htm[Java GSS Kerberos requirements]
 --
 
```

Comment:
The steps below are required if you want to use Kerberos Authentication via Kibana (which most people will want, but maybe not all).

I think we might want an introductory note here, but it's hard to say it refers to "the two steps below, but not the subsequent section ...".

Reply:
Alternatively, we could have indented them both under a "If you want to use Kerberos with Kibana:" step, but I think your suggestions are just as clear. Thanks!